sdk/keyvault/azure-keyvault-keys/tests/keys_preparer.py

```diff
@@ -113,12 +113,16 @@ def create_resource(self, name, **kwargs):
             parameters = VaultCreateOrUpdateParameters(location=self.location, properties=properties)
 
             self.management_client = self.create_mgmt_client(KeyVaultManagementClient)
+            try:
+                hashed_padding = str(hash(os.environ['RUN_IDENTIFIER'])).replace("-", "")[:4]
```

Comment:
One could get rid of the string replace using `abs` for a somewhat faster execution time

```
hashed_padding = str(abs(hash(os.environ['RUN_IDENTIFIER'])))[:4]
```